### ISSUE TITLE
x0vncserver: No reverse connection possible

### ISSUE THREAD

**Issue by robschi** [TigerVNC/tigervnc#697]
It is not possible to reverse connect to a running x0vncserver with "vncconfig -connect \<ip\>:5500".
It always shows the error message "**VNC extension on display :0**"

This is very unfortunate, because this would be a great way to give remote support to a friend over the internet.

**Comment by CendioOssman** [comment]
Right. `vncconfig` is just for `Xvnc`, and `x0vncserver` currently lacks this feature.
